mount/init.go

```diff
@@ -58,14 +58,17 @@ func InitializeMountNamespace(rootfs, console string, sysReadonly bool, mountCon
 		return fmt.Errorf("create device nodes %s", err)
```

Comment:
You are right about SetProcessLabel and ApplyProfile. I have to move them to init. I was aware but missed moving them back. I will post an updated patch with more comments. Thanks.

Sent from my iPhone

> On Jan 13, 2015, at 12:29 AM, Andrew Vagin <notifications@github.com> wrote:
> 
> In mount/init.go:
> 
> > @@ -58,14 +58,17 @@ func InitializeMountNamespace(rootfs, console string, sysReadonly bool, mountCon
> >  		return fmt.Errorf("create device nodes %s", err)
> @mrunalp Sorry, I don't get the idea with a setup process. Which actions can't be done from userns? Could you elaborate? Maybe you can add a comment before the SetupContainer() function.
> LoadContainerEnvironment(), apparmor.ApplyProfile(), label.SetProcessLabel() affect only a current process. Now we call them for the setup process and don't call for the init process. Where are these parameters applied to the init process?
> Thanks.
> 
> —
> Reply to this email directly or view it on GitHub.
>